### ISSUE TITLE
generated pleasew doesn't recognise FreeBSD

### ISSUE THREAD

**Issue by mcgaw** [thought-machine/please#2370]
FreeBSD is missing from this OS determination (fixed here):

```
# Find the os / arch to download. You can do this quite nicely with go env
# but we use this script on machines that don't necessarily have Go itself.
if [ "$OS" = "Linux" ]; then
    GOOS="linux"
elif [ "$OS" = "Darwin" ]; then
    GOOS="darwin"
elif [ "$OS" = "FreeBSD" ]; then
    GOOS="freebsd"
else
    echo -e >&2 "${RED}Unknown operating system $OS${RESET}"
    exit 1
fi
```
With this change I was able to use pleasew on FreeBSD.

**Comment by samwestmoreland** [comment]
Thanks for this! Would you like to open a pull request for this change so we can merge it? Otherwise I can go ahead and create one.

**Comment by mcgaw** [comment]
No problem! [MR 2372](https://github.com/thought-machine/please/pull/2372)

**Comment by samwestmoreland** [comment]
Closed by #2372
